src/index.test.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    AxeResults,
+    Result,
+    RunOptions,
+    TestEngine,
+    TestEnvironment,
+    TestRunner,
+} from 'axe-core';
+import * as fs from 'fs';
+import { axeToSarif } from './index';
+import { SarifLog } from './sarif/sarifLog';
+import { Run } from './sarif/sarifv2';
+
+describe('axe-sarf-converter integration test', () => {
+    let axeResultStub: AxeResults;
+    beforeEach(() => {
+        axeResultStub = {
+            toolOptions: {} as RunOptions,
+            testEngine: {} as TestEngine,
+            testRunner: {} as TestRunner,
+            testEnvironment: {} as TestEnvironment,
+            url: 'https://abc.com',
```

Comment:
Avoid using an uncontrolled 3rd party site; it implies that we're affiliated with them. Consider `http://example.com` instead (it's defined by standards for this purpose)